javacutil/src/main/java/org/checkerframework/javacutil/SystemUtil.java

```diff
@@ -225,4 +226,21 @@ public static int getJreVersion() {
         }
         return javaHome + File.separator + "lib" + File.separator + "tools.jar";
     }
+
+    /**
+     * Given an array and two values, creates an array with all the values. The two extra values are
+     * at the end of the returned array.
+     *
+     * @param array the array to extend
+     * @param elt1 the first element to add
+     * @param elt2 the second element to add
+     * @return a copy of {@code array}, with two elements added at the end
+     */
+    public static Object[] arrayPlusTwoElements(Object[] array, Object elt1, Object elt2) {
```

Comment:
This would be more general with a method type parameter `<T>` instead of `Object`.